### PR TITLE
enhance the instruction about the kafka exporter function.

### PR DESCRIPTION
**Pull request by Henry75m39**
A. change the instruction about how to activate export function: To activate the export program, you should set "${SW_exporter: nefault}" and "${SW_exporterENABLE_KAFKALOG: true}", configure the KAFKA server address.   B. change the config information code part.
add selector:
selector: ${SW_EXPORTER:default}

- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [ ] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

### CODE REVIEW

File: docs/en/setup/backend/exporter.md [hunk 1]
```diff
@@ -72,9 +72,10 @@ message SegmentObject {
 }
 ```
 
-To activate the exporter, you should set `${SW_EXPORTER_ENABLE_KAFKA_TRACE:true}` and config the Kafka server.
+To activate the exporter, you should set `${SW_EXPORTER:default}` and `${SW_EXPORTER_ENABLE_KAFKA_TRACE:true}`, configure the Kafka server address.
```
Comment (wu-sheng) [inline docs/en/setup/backend/exporter.md:75]:
```suggestion
To activate the exporter, you should set `${SW_EXPORTER:default}` and `${SW_EXPORTER_ENABLE_KAFKA_TRACE:true}`, configure the Kafka server addresses.
```

File: docs/en/setup/backend/exporter.md [hunk 2]
```diff
@@ -111,9 +112,10 @@ message LogData {
 }
 ```
 
-To activate the exporter, you should set `${SW_EXPORTER_ENABLE_KAFKA_LOG:true}` and config the Kafka server.
+To activate the exporter, you should set `${SW_EXPORTER:default}` and `${SW_EXPORTER_ENABLE_KAFKA_LOG:true}`, configure the Kafka server address.
```
Comment (wu-sheng) [inline docs/en/setup/backend/exporter.md:115]:
```suggestion
To activate the exporter, you should set `${SW_EXPORTER:default}` and `${SW_EXPORTER_ENABLE_KAFKA_LOG:true}`, configure the Kafka server addresses.
```

**Comment by wankai123** [comment]
Please help to enhance the gRPC exporter config part as well, thanks.

File: docs/en/setup/backend/exporter.md [hunk 3]
```diff
@@ -26,9 +26,10 @@ service MetricExportService {
 }
 ```
 
-To activate the exporter, you should set `${SW_EXPORTER_ENABLE_GRPC_METRICS:true}` and config the target gRPC server address.
+To activate the exporter, you should set `${SW_EXPORTER:default}` and `${SW_EXPORTER_ENABLE_GRPC_METRICS:true}`, configure the target gRPC server addresses.
```
Comment (wu-sheng) [inline docs/en/setup/backend/exporter.md:29]:
```suggestion
To activate the exporter, you should set `${SW_EXPORTER:default}` and `${SW_EXPORTER_ENABLE_GRPC_METRICS:true}`, configure the target gRPC server address.
```

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**